MoneySplitter.Win10/CustomControls/CollaboratorControl.xaml.cs

```diff
@@ -0,0 +1,50 @@
+﻿using MoneySplitter.Models;
+using System;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace MoneySplitter.Win10.CustomControls
+{
+    public sealed partial class CollaboratorControl : UserControl
+    {
+        public event EventHandler<CollaboratorModel> OnGiveCollaboratorButtonClick;
+        public event EventHandler<CollaboratorModel> OnRemindCollaboratorButtonClick;
+        public event EventHandler<CollaboratorModel> OnApproveCollaboratorButtonClick;
+
+        public CollaboratorModel ViewModel
+        {
+            get { return (CollaboratorModel)GetValue(ViewModelProperty); }
+            set { SetValue(ViewModelProperty, value); }
+        }
+
+
+        public static readonly DependencyProperty ViewModelProperty = DependencyProperty.Register(
+            "ViewModel",
+            typeof(CollaboratorModel),
+            typeof(CollaboratorControl),
+            null);
+
+        public CollaboratorControl()
+        {
+            InitializeComponent();
+            GiveCollaboratorButton.Content = Defines.Collaborator.ButtonContent.GIVE;
```

Comment:
We have XamlElementExtensions class for this.

Reply:
don't have